charts/appswitcher-server/templates/secrets.yaml

```
@@ -2,9 +2,9 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ include "appswitcher-server-helm-chart.fullname" . }}
+  name: {{ include "common.fullname" . }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)